gapic-generator-cloud/test/test_helper.rb

```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 $LOAD_PATH.unshift File.expand_path("../lib", __dir__)
+$LOAD_PATH.unshift ::File.expand_path("../../gapic-generator/test", __dir__)
```

Comment:
both this and the similar one in base generator are for the purpose of sharing the test resources for the parameter parsing:

require "gapic/schema/api_test_resources"

other than `test_helper.rb` there are no files that share the name but I see how this might lead to confusion. 
What would be a better way to share the resources?

Reply:
moved the shared test resources to shared instead